tutorials/02-functions/03-currying-and-partial-application.md

```diff
@@ -5,9 +5,11 @@ Read on, my friends.
 
 ## Partial application
 
-Partial application is a technique that allows us to pre-fill arguments to a function. Earlier, we discussed how you can use `bind()` on functions in JavaScript to explicitly set the value of `this`. Like `call()`, `bind()` takes additional arguments and will set those arguments on the function it returns.
+Partial application is a technique that allows us to pre-fill arguments to a function. [In the previous section][What is this], we discussed how you can use `bind()` on functions in JavaScript to explicitly set the value of `this`. Like `call()`, `bind()` takes additional arguments and will set those arguments on the function it returns.
```

Comment:
Would this be a suitable solution for #28? Since the tutorials are numbered, I think having this in `03-currying-and-partial-application` is fine since it follows `02-what-is-this`.